src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -73,6 +93,53 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
 
     return (
       <>
+        {(() => {
```

Comment:
similar to the others in CreatePipelineRun, `return null` not required

Reply:
Thanks. I missed that one 👍